src/Pub/API/DTOs/ProjectDto.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace API.DTOs
+{
+    public class ProjectDto
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public DateTime LaunchDate { get; set; }
```

Comment:
Let's use DateTimeOffset here.
```suggestion
        public DateTimeOffset LaunchDate { get; set; }
```